content/en/monitors/create/types/error_tracking.md

```diff
@@ -0,0 +1,126 @@
+
+1. Select or input a custom time period for the monitor to consider an issue as new after its first occurrence. The selected threshold is evaluated in the given time frame. After the specific time period, the monitor stops alerting and turns green.
+2. Select **Web and Mobile Apps** or **Backend Services** and choose to monitor over a count or [measure][1]. 
+   - Monitor over a unique count based on the issue ID. 
+   - Monitor over a measure. If you select a measure, the monitor alerts over the numerical value of the RUM or APM facet (similar to a metric monitor). Select an aggregation type (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
+3. Construct a search query using the same logic as a [RUM Explorer search][2] or [APM Explorer search][3] for the issues’ error occurrences.
+4. Configure the alerting grouping strategy (optional).
+   - Simple-Alert: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. If the query has a `group by` and you select **Simple-Alert**, you get one alert when one or multiple groups’ values breach the threshold. You may use this strategy to reduce notification noise.
+   - Multi-Alert: Multiple alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you can group a query by `@browser.name` to receive a separate alert for each browser when the number of errors is high.
+
+Triggers when the number of errors is `above` or `above or equal to`. An alert is triggered whenever a metric crosses a threshold.
+
+- The threshold during the last `5 minutes`, `15 minutes`. `1 hour`, or `custom` to set a value between 5 minutes and 48 hours. 
+- Alerting threshold `<NUMBER>`.
+- Warning threshold `<NUMBER>`.
+
+[1]: /real_user_monitoring/explorer/?tab=measures#setup-facets-measures
+[2]: /real_user_monitoring/explorer/search/
+[3]: /tracing/trace_explorer/?tab=listview#filtering
+{{% /tab %}} 
+{{< /tabs >}}
+
+#### Advanced Alert Conditions
+
+For more information about advanced alert options such as evaluation frequency, see [Configure Monitors][5].
+
+### Notifications
+
+To display triggering tags in the notification title, click **Include triggering tags in notification title**.
+
+For more information about the **Notify your team** and **Say what’s happening** sections, see [Notifications][6].
+
+#### Breaching Values Toplist
+
+When an Error Tracking monitor is triggered, values can be added to the notification message. 
+
+| Monitor Over     | Can be added to notification message                                                               |
+|------------------|----------------------------------------------------------------------------------------------------|
+| Facet or measure | Grouped: The top 10 facet or measure values.<br>Ungrouped: The top 10 facet or measure values. |
```

Comment:
What's the difference between "grouped" and "ungrouped". They both say "The top 10 facet or measure values". Maybe this table can be replaced by another sentence or phrase in line 103?

Reply:
```suggestion
```